src/app/search/search-notebook-table/search-notebook-table.component.html

```diff
@@ -13,8 +13,17 @@
               notebook?.source.repository +
               (notebook?.source.workflowName ? '/' + notebook?.source.workflowName : '')
           }}</a>
-          <div class="truncate-text-2 size-small" *ngIf="notebook.source.topicAutomatic" [matTooltip]="notebook.source.topicAutomatic">
-            {{ notebook.source.topicAutomatic }}
+          <div class="truncate-text-2 size-small" [matTooltip]="notebook?.source.topicAutomatic" matTooltipPosition="left">
+            {{ notebook?.source.topicAutomatic }}
+            <span *ngIf="notebook?.source.topicSelection === TopicSelectionEnum.AI">
```

Comment:
You could remove this outer span and move the `ngIf` condition to the inner span with the tooltip. Ditto for the other search tables